docs/concepts/storage/persistent-volumes.md

```diff
@@ -417,6 +429,86 @@ spec:
 
 `PersistentVolumes` binds are exclusive, and since `PersistentVolumeClaims` are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
 
+## Raw Block Volume Support
+
+Static provisioning support for Raw Block Volumes is included as an alpha feature for v1.9. With this change are some new API labels that need to be used to facilitate this functionality. Currently, Fibre Channel is the only supported plugin for this feature.
+
+### Persistent Volumes using a Raw Block Volume
+```
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: block-pv
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block
+  persistentVolumeReclaimPolicy: Retain
+  fc:
+    targetWWNs: ["50060e801049cfd1"]
+    lun: 0
+    readOnly: false
+```
+### Persistent Volume Claim requesting a Raw Block Volume
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: block-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block
+  resources:
+    requests:
+      storage: 10Gi
+```
+### Pod specification adding Raw Block Device path in container
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-with-block-volume
+spec:
+  containers:
+    - name: fc-container
+      image: fedora:26
+      command: ["/bin/sh", "-c"]
+      args: [ "tail -f /dev/null" ]
+      volumeDevices:
+        - name: data
+          devicePath: /dev/xvda
+  volumes:
+    - name: data
+      persistentVolumeClaim:
+        claimName: block-pvc
+```
+#### Volume Devices
+
+Since we aren't mounting a filesystem, when using a raw block device for a pod we specify the device path in the container rather than a mount path.
```

Comment:
```
**Note:** When mounting a raw block device for a Pod instead of a file system, specify the device path in the container instead of a mount path.
{: .note}
```
So:
> **Note:** When mounting a raw block device for a Pod instead of a file system, specify the device path in the container instead of a mount path.
> {: .note}

Reply:
I'd prefer to use 'attaching' for raw block device. Mounting a raw block device is little bit strange expression.

---

When attaching a raw block device for a Pod instead of mounting a file system, specify the device path in the container instead of a mount path.

Reply:
But will attaching be confused with volume attaching

Reply:
In the docker document, they use 'add' for device. Do we use same term or mounting is ok for this case?

`--device |   | Add a host device to the container`

Reply:
Add is probably the best option